web/src/stories/GenerateSupportBundleModal.wip.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { MemoryRouter } from "react-router-dom";
+import GenerateSupportBundleModal from "@src/components/troubleshoot/GenerateSupportBundleModal";
+
+// TODO: update after adding fetch mock support to storybook
```

Comment:
LMK if this is available. If not we can work on it soon- should be a blog or something we can riff off of.

Reply:
not available yet. :)